BlueprintUI/Sources/Layout/Overlay.swift

```diff
@@ -10,28 +10,39 @@ import UIKit
 public struct Overlay: Element {
 
     /// All elements displayed in the overlay.
-    public var elements: [Element]
+    public var children: [Child]
 
     /// Creates a new overlay with the provided elements.
     public init(
         elements: [Element] = [],
         configure: (inout Overlay) -> Void = { _ in }
     ) {
-        self.elements = elements
+        children = elements.map { Child(element: $0) }
         configure(&self)
     }
 
     /// Adds the provided element to the overlay.
+    @available(*, deprecated, renamed: "add(child:)")
```

Comment:
IMO keeping this and not deprecating it is probably fine; since not everyone needs/wants keys?

Reply:
(Eg, so the shorter method is probably fine to avoid a ton of deprecation warnings)

Reply:
meh, stacks use `add(child:)` and I kinda like the consistency? no strong opinion though

Reply:
I thought it was confusing to have both `add(_:)` and `add(child:)`, and the latter is consistent with other stacks.